website/docs/guides/blocks-render-block-view-helper.md

```diff
@@ -11,7 +11,7 @@ Locate and return template part with passed attributes for a block.
 
 Inside your block view PHP file you'd write:
 
-```php
+```js
```

Comment:
Because with PHP code  the formatting looks like this:
![Screenshot 2020-06-08 at 09 50 49](https://user-images.githubusercontent.com/23283324/84005583-a1d76a00-a96d-11ea-81b1-636e3402b58b.png)

and with JS it looks like this:
![Screenshot 2020-06-08 at 09 50 30](https://user-images.githubusercontent.com/23283324/84005602-aa2fa500-a96d-11ea-947f-f7fe5077a066.png)

Reply:
> Because with PHP code the formatting looks like this:
> ...
> and with JS it looks like this:
> ...

That's equal parts sad and funny 😅